UnitsNet/GeneratedCode/Quantities/Acceleration.g.cs

```diff
@@ -259,6 +259,46 @@ public Acceleration(double value, UnitSystem unitSystem)
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.StandardGravity, quantity => quantity.ToUnit(AccelerationUnit.StandardGravity));
+            
+            // Register in unit converter: BaseUnit <-> BaseUnit
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity);
+
+            // Register in unit converter: AccelerationUnit -> BaseUnit
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.CentimeterPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.DecimeterPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.FootPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.InchPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.KilometerPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.KnotPerHour, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.KnotPerMinute, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.KnotPerSecond, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MicrometerPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MillimeterPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MillistandardGravity, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.NanometerPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+            unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.StandardGravity, AccelerationUnit.MeterPerSecondSquared, quantity => quantity.ToBaseUnit());
+        }
```

Comment:
I like the new structure ♥

Reply:
A minor idea, but since we are just about always adding two-way conversions, could `SetConversionFunction` have an overload method to define both ways in one call?

Pretty much just an extra conversion function parameter if I am reading it right.

Reply:
Could be named `SetTwoWayConversionFunctions`

Reply:
> A minor idea, but since we are just about always adding two-way conversions, could `SetConversionFunction` have an overload method to define both ways in one call?
> 
> Pretty much just an extra conversion function parameter if I am reading it right.

I tried adding an overload, but IMHO the lines became a bit more "mentally taxing" (even more so than it already is). It is certainly shorter in height though:
```csharp
public void SetConversionFunction<TQuantity>(Enum from, Enum to, ConversionFunction<TQuantity> conversionFunction, ConversionFunction<TQuantity> reverseConversionFunction)
    where TQuantity : IQuantity
{
}
```
```csharp
/// <summary>
/// Registers the default conversion functions in the given <see cref="UnitConverter"/> instance.
/// </summary>
/// <param name="unitConverter">The <see cref="UnitConverter"/> to register the default conversion functions in.</param>
internal static void RegisterDefaultConversions(UnitConverter unitConverter)
{
    // Register in unit converter: BaseUnit <-> BaseUnit
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity);

    // Register in unit converter: BaseUnit -> AccelerationUnit
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.CentimeterPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.CentimeterPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.DecimeterPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.DecimeterPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.FootPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.FootPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.InchPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.InchPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KilometerPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.KilometerPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KnotPerHour, quantity => quantity.ToUnit(AccelerationUnit.KnotPerHour), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KnotPerMinute, quantity => quantity.ToUnit(AccelerationUnit.KnotPerMinute), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KnotPerSecond, quantity => quantity.ToUnit(AccelerationUnit.KnotPerSecond), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MicrometerPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.MicrometerPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MillimeterPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.MillimeterPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MillistandardGravity, quantity => quantity.ToUnit(AccelerationUnit.MillistandardGravity), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.NanometerPerSecondSquared, quantity => quantity.ToUnit(AccelerationUnit.NanometerPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.StandardGravity, quantity => quantity.ToUnit(AccelerationUnit.StandardGravity), quantity => quantity.ToBaseUnit());
}
```

Breaking it into multiple lines gets even worse:
```csharp
/// <summary>
/// Registers the default conversion functions in the given <see cref="UnitConverter"/> instance.
/// </summary>
/// <param name="unitConverter">The <see cref="UnitConverter"/> to register the default conversion functions in.</param>
internal static void RegisterDefaultConversions(UnitConverter unitConverter)
{
    // Register in unit converter: BaseUnit <-> BaseUnit
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MeterPerSecondSquared, quantity => quantity);

    // Register in unit converter: BaseUnit -> AccelerationUnit
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.CentimeterPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.CentimeterPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.DecimeterPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.DecimeterPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.FootPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.FootPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.InchPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.InchPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KilometerPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.KilometerPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KnotPerHour,
        quantity => quantity.ToUnit(AccelerationUnit.KnotPerHour), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KnotPerMinute,
        quantity => quantity.ToUnit(AccelerationUnit.KnotPerMinute), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.KnotPerSecond,
        quantity => quantity.ToUnit(AccelerationUnit.KnotPerSecond), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MicrometerPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.MicrometerPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MillimeterPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.MillimeterPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.MillistandardGravity,
        quantity => quantity.ToUnit(AccelerationUnit.MillistandardGravity), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.NanometerPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.NanometerPerSecondSquared), quantity => quantity.ToBaseUnit());
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, AccelerationUnit.StandardGravity,
        quantity => quantity.ToUnit(AccelerationUnit.StandardGravity), quantity => quantity.ToBaseUnit());
}
```

Reply:
Got it. I agree, the line became tough to read. The best I could come up with was:
```cs
    unitConverter.SetConversionFunction<Acceleration>(AccelerationUnit.MeterPerSecondSquared, 
        AccelerationUnit.CentimeterPerSecondSquared,
        quantity => quantity.ToUnit(AccelerationUnit.CentimeterPerSecondSquared), 
        quantity => quantity.ToBaseUnit());
```

Doesn't save vertical space though, so I don't know. I don't think it helps on neither the issue we are trying to fix nor readability. The code is also generated anyhow. LGTM!